cmake/init_target.cmake

```
@@ -75,7 +75,7 @@ function(init_target target_name) # init_target(my_target folder_name)
             endif()
         endif()
 
-        if (CMAKE_SYSTEM_PROCESSOR MATCHES "i686.*|i386.*|x86.*")
+        if (CMAKE_SYSTEM_PROCESSOR MATCHES "i686.*|i386.*")
```

```suggestion
        if (is_x86 AND CMAKE_SYSTEM_PROCESSOR MATCHES "i686.*|i386.*|x86.*")
```
Just like in https://github.com/telegramdesktop/tdesktop/blob/f1cdc7e3f9c67b0c96ed9172b89e6c11e0c206a5/Telegram/cmake/lib_tgvoip.cmake#L167

---

Done.